src/views/open-api-docs-view/components/open-api-operations/index.tsx

```diff
@@ -29,21 +33,31 @@ export function OpenApiOperations({
 				.map((group) => group.items)
 				.flat()
 				.map((operation: OperationProps) => {
+					const operationIdWithBreaks = addWordBreaks(
+						splitOnCapitalLetters(operation.operationId)
+					)
+
 					return (
 						<div key={operation.operationId}>
 							<OperationSections
 								headerSlot={
 									<OperationHeader
 										className={s.header}
 										slug={operation.slug}
-										headerText={operation.operationId}
+										headingAriaLabel={operation.operationId}
+										headingSlot={operationIdWithBreaks}
```

Comment:
This is a slight tangent, but do we have access to a more human-friendly value, rather than `operationId`? 

The sibling `summary` field is what should accomplish this for us.
- https://swagger.io/specification#operation-object

And if HCP specs are missing this field, I think we should have some say in tasking them with populating the field.
- https://github.com/hashicorp/hcp-specs/blob/main/specs/cloud-vault-secrets/preview/2023-06-13/hcp.swagger.json#L23C9-L23C9

Reply:
Great call, very on board with this! Made a separate task for this: 🎟️ [Use operation "summary" rather than "operationId" as more human-readable operation title](https://app.asana.com/0/1204678746647847/1205316467934575/f). Will try to get this in earlier rather than later to make it clear we're going more strongly in the direction of "follow the spec" 👍

Reply:
What am i saying i'm already got `summary` available, will just do it in this PR!